DocumentUnderstandingProcess/UiPath.Template.DocumentUnderstandingProcess.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
   <metadata>
     <id>UiPath.Template.DocumentUnderstandingProcess</id>
-    <version>21.4.4-preview-test</version>
+    <version>21.4.4-preview</version>
```

Comment:
check if this is the correct version ? 
we had a talk about 1.0.0-preview

Reply:
After some discussion in-house, we agreed to move with versioning identical to the UiPath Studio